tools/raml2html/src/Twig/Extension/HashExtension.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\Raml2Html\Twig\Extension;
+
+use RuntimeException;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class HashExtension extends AbstractExtension
+{
+    private $hashes = [];
```

Comment:
is it possible for us to force higher PHP version here and use its features, like strict type in this case? I'd go for the latest PHP 8.

Reply:
@alongosz I moved to PHP8.1+ in #1871. I can take care of this there.

Reply:
Technically possible, it's an internal tool.

Well, copied from an open source library afaik.

However, I'm not sure what versions of PHP our doc team have, and I'm not sure if we can just force PHP 8. PHP 7.4 would be more suitable?

Reply:
I propose #1898 as quick fix for 7.4 as it doesn't work with PHP 7.2 anymore.

For PHP 8.1+, I'll make a survey.

Reply:
We're all good with at least PHP 8.1

Reply:
Let's merge this w/ 7.4 and I'll continue my 8.1 PR afterward.